evm/utils.js

```diff
@@ -636,18 +636,22 @@ const mainProcessor = async (options, processCommand, save = true, catchErr = fa
         throw new Error('Environment was not provided');
     }
 
-    if (!options.chainName && !options.chainNames) {
-        throw new Error('Chain names were not provided');
-    }
-
     printInfo('Environment', options.env);
 
     const config = loadConfig(options.env);
-    let chains = options.chainName ? [options.chainName] : options.chainNames.split(',');
     const chainsToSkip = (options.skipChains || '').split(',').map((str) => str.trim().toLowerCase());
 
+    let chains = [];
+
     if (options.chainNames === 'all') {
         chains = Object.keys(config.chains);
+        chains = chains.filter((chain) => config.chains[chain].chainType === 'evm');
```

Comment:
```suggestion
        chains = chains.filter((chain) => !config.chains[chain].chainType || config.chains[chain].chainType === 'evm');
```
treat it as evm by default to avoid breaking any local configs being used. we can remove this after a while